src/main/java/tp/DeleteDoctorCommand.java

```diff
@@ -15,8 +15,13 @@ public String execute(DoctorList doctorList, PatientList patientList,
                         PatientStorage patientStorage,
                         AppointmentStorage appointmentStorage) throws IHospitalException {
         Doctor curr = doctorList.deleteDoctor(index);
-        return String.format(boundary + "Noted. I've removed this doctor:" + curr
-                + "\n" + "Now you have " + doctorList.getSize()
-                + " doctors in the system." + System.lineSeparator() + boundary);
+        if (curr == null){
```

Comment:
<img width="967" alt="Screen Shot 2022-04-05 at 11 53 02 PM" src="https://user-images.githubusercontent.com/69306031/161795707-0c1e2c75-2113-4c5b-a53a-7dabdc907c57.png">
You will need to resolve the conflicts before merging.

Reply:
should pass the checkstyles

Reply:
should resolvethe merge conflict